alphacsc/convolutional_dictionary_learning.py

```diff
@@ -293,7 +293,7 @@ def uv_hat_(self):
         if self._D_hat.ndim == 3:
```

Comment:
Can you update the comment to describe the convention that we take (peak of v > 0)?

Reply:
I updated the `uv_hat_` method docstring.